test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -762,7 +762,7 @@ public void SelectionButtonShouldBeDisabledAfterOpeningNewDocument()
 
         [Test]
         [Category("RegressionTests")]
-        [TestModel(@".\Samples\DynamoSample.rvt")]
+        [TestModel(@".\Samples\DynamoSample_2020.rvt")]
```

Comment:
Is this for speeding the test up?

Reply:
No, I just found that the DynamoSample.rvt file was deleted by last PR https://github.com/DynamoDS/DynamoRevit/pull/2387, and DynamoRevit would get Sample file from nuget.org.
I'm sorry for this missing.